static/app/views/dashboardsV2/sortableWidget.tsx

```diff
@@ -73,6 +75,7 @@ function SortableWidget(props: Props) {
     currentWidgetDragging,
     showContextMenu: true,
     isPreview,
+    tableItemLimit: TABLE_ITEM_LIMIT,
```

Comment:
Can we add this under the feature flag, e.g. after line 84? It's going to make widgets on the dndkit dashboard really long!